planning/scenario_planning/lane_driving/motion_planning/surround_obstacle_checker/surround_obstacle_checker-design.ja.md

```diff
@@ -46,20 +46,20 @@ Stop condition の項で述べたように、状態によって障害物判定
 
 ### Input
 
-| Name                                     | Type                                                | Description          |
-| ---------------------------------------- | --------------------------------------------------- | -------------------- |
-| `~/input/trajectory`                     | `autoware_planning_msgs::msg::Trajectory`           | Reference trajectory |
-| `/sensing/lidar/no_ground/pointcloud`    | `sensor_msgs::msg::PointCloud2`                     | No ground pointcloud |
-| `/perception/object_recognition/objects` | `autoware_perception_msgs::msg::DynamicObjectArray` | Dynamic objects      |
-| `/localization/twist`                    | `geometry_msgs::msg::TwistStamped`                  | Current twist        |
-| `/tf`                                    | `tf2_msgs::msg::TFMessage`                          | TF                   |
-| `/tf_static`                             | `tf2_msgs::msg::TFMessage`                          | TF static            |
+| Name                                     | Type                                                     | Description          |
+| ---------------------------------------- | -------------------------------------------------------- | -------------------- |
+| `~/input/trajectory`                     | `autoware_auto_planning_msgs::msg::Trajectory`           | Reference trajectory |
+| `/sensing/lidar/no_ground/pointcloud`    | `sensor_msgs::msg::PointCloud2`                          | No ground pointcloud |
+| `/perception/object_recognition/objects` | `autoware_auto_perception_msgs::msg::DynamicObjectArray` | Dynamic objects      |
+| `/localization/twist`                    | `geometry_msgs::msg::TwistStamped`                       | Current twist        |
```

Comment:
`localization/twist?`

Reply:
Ah, this file was supposed to be removed, but it was decided no to be removed. So I'll fix. Thanks

Reply:
https://github.com/tier4/AutowareArchitectureProposal.iv/pull/576/commits/cfb7d2d87ec9f4479f6ee74422af82100868a3dd
Done